.travis.yml

```diff
@@ -7,6 +7,10 @@ before_install:
   # Install the coverage utility and coveralls reporting utility
   - pip install coverage
   - pip install coveralls
+  # FIXME (18-Apr-2016) cnx-easybake requires an unreleased package, cssselect2
+  - pip install git+https://github.com/SimonSapin/cssselect2.git#egg=cssselect2
```

Comment:
I've forked cssselect2 into the Connexions space, in prep for taking it over, so this could switch to:
git+https://github.com/Connexions/cssselect2.git#egg=cssselect2